pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -469,7 +471,17 @@ func (r *SubnetPortReconciler) CheckAndGetSubnetPathForSubnetPort(ctx context.Co
 			log.Error(err, "failed to get NSX subnet by subnet CR UID", "subnetList", subnetList)
 			return
 		}
-		subnetPath = *subnetList[0].Path
+		nsxSubnet := subnetList[0]
+		totalIP := int(*nsxSubnet.Ipv4SubnetSize)
```

Comment:
I would prefer to use totalIP as a field of `CountWithMark` which is a one-time set value when we create the object for a new subnet, then we can consume it any time we need to allocate a new port with the diff.

Reply:
Updated, thanks